testkit/src/main/scala/prometheus4cats/testkit/CallbackRegistrySuite.scala

```diff
@@ -424,4 +426,118 @@ trait CallbackRegistrySuite[State] extends RegistrySuite[State] { self: CatsEffe
         }
     }
   }
+
```

Comment:
I _think_ this is missing a case where you have a callback of the same name with different label names. This _should_ be permissable now, thanks to the hacking 😂

Reply:
Yeah, I wondered about that and then forgot about it again 🤦 Good spot 👍